core/consensus/babe/impl/epoch_storage_impl.cpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "consensus/babe/impl/epoch_storage_impl.hpp"
+
+#include "scale/scale.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::consensus, EpochStorageError, e) {
+  using E = kagome::consensus::EpochStorageError;
+
+  switch (e) {
+    case E::EPOCH_DOES_NOT_EXIST:
+      return "Requested epoch does not exist";
+  }
+}
+
+namespace kagome::consensus {
+
+  const static auto EPOCH_PREFIX =
```

Comment:
Hm, why does it have to be static if it's in a cpp file?

Reply:
Could be just global variable as well. Does not matter that much, will update